MediaBrowser.XbmcMetadata/Savers/EpisodeNfoSaver.cs

```diff
@@ -14,43 +14,43 @@ namespace MediaBrowser.XbmcMetadata.Savers
 {
     public class EpisodeNfoSaver : BaseNfoSaver
     {
-        protected override string GetLocalSavePath(BaseItem item)
+        public EpisodeNfoSaver(IFileSystem fileSystem, IServerConfigurationManager configurationManager, ILibraryManager libraryManager, IUserManager userManager, IUserDataManager userDataManager, ILogger logger)
+            : base(fileSystem, configurationManager, libraryManager, userManager, userDataManager, logger)
         {
-            return Path.ChangeExtension(item.Path, ".nfo");
         }
 
+        private readonly CultureInfo _usCulture = new CultureInfo("en-US");
+
+        /// <inheritdoc />
+        protected override string GetLocalSavePath(BaseItem item)
+            => Path.ChangeExtension(item.Path, ".nfo");
+
+        /// <inheritdoc />
         protected override string GetRootElementName(BaseItem item)
-        {
-            return "episodedetails";
-        }
+            => "episodedetails";
 
+        /// <inheritdoc />
         public override bool IsEnabledFor(BaseItem item, ItemUpdateType updateType)
-        {
-            if (!item.SupportsLocalMetadata)
-            {
-                return false;
-            }
-
-            return item is Episode && updateType >= MinimumUpdateType;
-        }
+            => !item.SupportsLocalMetadata && item is Episode && updateType >= MinimumUpdateType;
```

Comment:
still need to invert this one

Reply:
@Bond-009 Thoughts?

Reply:
It seems to match the old code now

Reply:
@joshuaboniface this was definitely not fixed, the opposite change is in multiple locations. Either this section is wrong or the other sections like [this](https://github.com/Bond-009/jellyfin/blob/3eca8b9c98e5ad0ad9cefd85f27439f05f5bd6b5/MediaBrowser.XbmcMetadata/Savers/AlbumNfoSaver.cs#L39) one are wrong.

Reply:
@dkanada Sorry, fixed in #1775